raiden/tests/integration/long_running/test_stress.py

```diff
@@ -336,6 +340,7 @@ def assert_channels(
         )
 
 
+@pytest.mark.skip(reason="flaky, see https://github.com/raiden-network/raiden/issues/4803")
```

Comment:
Did you hit this in this PR? It seems like this doesn't happen often.

Reply:
Yes. It also happened in the build of #5266, and in about half the times I ran the stress test locally on `develop`. Not sure why it should suddenly happen more often though.